api/src/main/java/org/apache/iceberg/exceptions/AlreadyExistsException.java

```diff
@@ -19,14 +19,18 @@
 
 package org.apache.iceberg.exceptions;
 
+import com.google.errorprone.annotations.FormatMethod;
```

Comment:
Format Method allows for a compile time check that the format strings being used in String.Format are actually valid given the number of arguments passed to the method. This helps eliminate ignored variables or under-defined strings at compile time.

Reply:
Question: is this something that we should shade (possibly in a follow up PR)? I know that normally we shade thing like guava etc that the user might have on their classpath. Is it reasonable to expect that behavior here?

I don't think this needs to be addressed in this PR, but something that crossed my mind as I'm re-reviewing this.

Reply:
Oh. Actually upon further inspection I can see that this function `bindLiterralOperation` is only called once in this class (and in the entire code base currently) and there's a null check prior to calling it. So shouldn't be a concern at the moment 👍